### PR TITLE
Issue/20 exception logger

### PR DESCRIPTION
**Pull request by aurelenc**

### CODE REVIEW

File: Src/Core/Logger.hpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+** File description:
+** Logger
+*/
+
+#ifndef LOGGER_HPP_
+#define LOGGER_HPP_
+
+class ArcadeException;
+
+class Logger {
```
Comment (Cavonstavant) [inline Src/Core/Logger.hpp:13]: Add /// \brief documentation to class

**Review by Cavonstavant — CHANGES_REQUESTED** [review]
Class logger needs a short description

**Review by pariciadra — APPROVED** [review]
Et les 12h de sommeil aurele ?

**Review by Cavonstavant — APPROVED** [review]
LGTM!

**Pull request merged**